src/Microsoft.AspNet.Mvc.Core/FilterActionInvoker.cs

```diff
@@ -219,7 +368,36 @@ private IFilter[] GetFilters()
         private async Task InvokeAllActionFiltersAsync()
         {
             _cursor.SetStage(FilterStage.ActionFilters);
+
+            Debug.Assert(_resourceExecutingContext != null);
+            var bindingContext = new ActionBindingContext()
+            {
+                InputFormatters = _resourceExecutingContext.InputFormatters,
+                InputFormatterSelector = _resourceExecutingContext.InputFormatterSelector,
+                MetadataProvider = _modelMetadataProvider,
+                ModelBinder = new CompositeModelBinder(_resourceExecutingContext.ModelBinders),
+                ValidatorProvider = new CompositeModelValidatorProvider(_resourceExecutingContext.ValidatorProviders),
+            };
+
+            var valueProviders = new List<IValueProvider>();
```

Comment:
Added a static factory method for this. Due to the layer of MVC/ModelBinding, you still have to create a VPFContext.